packages/create-next-app/create-app.ts

```diff
@@ -126,63 +125,37 @@ export async function createApp({
       console.log()
       await downloadAndExtractExample(root, example)
     }
+  } else {
+    console.log('Downloading default files. This might take a moment.')
+    console.log()
 
-    // Copy our default `.gitignore` if the application did not provide one
-    const ignorePath = path.join(root, '.gitignore')
-    if (!fs.existsSync(ignorePath)) {
-      fs.copyFileSync(
-        path.join(__dirname, 'templates', 'default', 'gitignore'),
-        ignorePath
-      )
-    }
+    await downloadAndExtractExample(root, 'default')
 
-    console.log('Installing packages. This might take a couple of minutes.')
-    console.log()
+    const packageJson = JSON.parse(
+      fs.readFileSync(path.join(root, 'package.json'), 'utf8')
+    )
+    packageJson.name = appName
 
-    await install(root, null, { useYarn, isOnline })
-    console.log()
-  } else {
-    const packageJson = {
-      name: appName,
-      version: '0.1.0',
-      private: true,
-      scripts: { dev: 'next dev', build: 'next build', start: 'next start' },
-    }
     fs.writeFileSync(
       path.join(root, 'package.json'),
       JSON.stringify(packageJson, null, 2) + os.EOL
     )
+  }
 
-    console.log(
-      `Installing ${chalk.cyan('react')}, ${chalk.cyan(
-        'react-dom'
-      )}, and ${chalk.cyan('next')} using ${displayedCommand}...`
+  // Copy our default `.gitignore` if the application did not provide one
+  const ignorePath = path.join(root, '.gitignore')
+  if (!fs.existsSync(ignorePath)) {
+    fs.copyFileSync(
+      path.join(__dirname, 'templates', 'default', 'gitignore'),
```

Comment:
I wasn't quite sure what to do about this `gitignore` copying so I left it in place in `templates/default` so it can be copied/renamed during both flows (example & default). 

It didn't seem worth it to also store it in the `examples/default` folder because then for non "default" create-next-app runs it would have to then pull that `default` example down just to grab the `gitignore`.